packages/perseus/src/__tests__/server-item-renderer.test.tsx

```diff
@@ -118,31 +118,6 @@ describe("server item renderer", () => {
         expect(screen.getByRole("textbox")).toBeVisible();
     });
 
-    it("should be invalid if no input provided", async () => {
```

Comment:
I don't like deleting tests, but these seem to be specifically focused on testing SIR's ability to score.